docs/spaces.rst

```diff
@@ -36,7 +36,7 @@ that do not generate *standardized* coordinate spaces:
   * ``T1w`` or ``anat``: can be used to resample data into the individual's anatomical
     reference (e.g., the T1w image).
 
-  * ``func``,``run``, ``aslref`` or ``sbref`` can be used to
+  * ``func``,``run``, ``asl`` or ``sbref`` can be used to
```

Comment:
lines 39-42 are a little confusing. can those calls be used to generate the data after slice timing, head motion etc are computed? or after CBF is computed and slice timing, head motion etc are corrected for?